src/main/java/com/patsi/utils/GenerateIntHelper.java

```diff
@@ -0,0 +1,9 @@
+package com.patsi.utils;
+
+import java.io.IOException;
+
+public class GenerateIntHelper {
+    public static double generateRandomInt(int min, int max) {
+        return Math.floor(Math.random() * (max - min + 1) + min);
```

Comment:
wrap the Math.random() with a separated method

Reply:
fixed